sharding/collation.go

```diff
@@ -174,3 +204,19 @@ func DeserializeBlobToTx(serialisedBlob []byte) (*[]*types.Transaction, error) {
 
 	return &txs, nil
 }
+
+// Chunks is a wrapper around a chunk array to implement DerivableList,
+// which allows us to Merklize the chunks into the chunkRoot.
+type Chunks []byte
+
+// Len returns the number of chunks in this list.
+func (ch Chunks) Len() int { return len(ch) }
+
+// GetRlp returns the RLP encoding of one chunk from the list.
+func (ch Chunks) GetRlp(i int) []byte {
+	bytes, err := rlp.EncodeToBytes(ch[i])
+	if err != nil {
+		panic(err)
```

Comment:
Why panic? Why not just return ([]byte, error) and handle it somewhere?

Reply:
I'm using the ```DerivableList`` interface so that I can use the ```DeriveSha``` function, but I could always write a similar function specific to hashing for the chunk root.

Reply:
Yikes, why would we panic here? This will shutdown terminate the running process immediately. 

Please remove